Mathlib/Topology/DiscreteSubset.lean

```diff
@@ -145,4 +145,16 @@ lemma mem_codiscrete_subtype_iff_mem_codiscreteWithin {S : Set X} {U : Set S} :
     exact tendsto_nhdsWithin_of_tendsto_nhds_of_eventually_within _
       continuous_subtype_val.continuousWithinAt <| eventually_mem_nhdsWithin.mono (by simp)
 
+/-- If a set is codiscrete within `U`, then it is codiscrete within any subset of `U`. -/
+lemma Filter.codiscreteWithin.mono {X : Type u_1} [TopologicalSpace X] {U₁ U₂ : Set X}
```

Comment:
I think you should move this up between `mem_codiscreteWithin_accPt` and `Filter.codiscrete` as the remainding lemmas then all talk about `Filter.codiscrete` where as yours doesn't need that definition.

Reply:
Implemented.